content/posts/may-26-plugin-update.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "May 26th update to `mattermost-plugin-ai`"
+date: 2023-05-26T23:43:40+00:00
+draft: false
+---
+
+We are thrilled to announce an update to our `mattermost-plugin-ai`! This release brings a fresh new interface and enhancements to improve your experience with the AI Assistant. Let's explore the details: 
+
+✨ **New Interface for the AI Assistant**: Say hello to our revamped AI Assistant interface! We have given it a makeover to provide a more better user experience.
+
+🤖 **Renamed as AI Assistant**: We have renamed the LLM Bot to AI Assistant for better clarity. Now, you can easily call upon the AI Assistant by mentioning `@ai` instead of `@llmbot`.
+
+🎨 **Updated Profile Icon**: The AI Assistant now sports a brand new profile icon, giving it a fresh and distinctive look.
+
+👍👎 **Enhanced Feedback Reactions**: The thumbs-up and thumbs-down reactions have received a visual improvement and send feedback to the AI Assistant.
+
+🔠 **Font Reverted to Regular**: We have reverted from using a monospaced font to a regular font. However, don't worry! We are working on introducing additional controls to differentiate the AI Assistant from regular users so there is no confusion. 
+
+AI open source technologies continue to evolve, empowering us to create intelligent and collaborative tools that augment human capabilities. Let's embrace the potential of AI and open source to shape a future where intelligent assistants empower us to achieve greater productivity and success.
```

Comment:
@azigler what are your thoughts on adding a CTA, e.g. to the GitHub repo or the community channel?

Reply:
@jasonblais Great idea! I updated the blog post to use @asaadmahmood's new metadata format, added CTAs for the channel and forums, and changed the repo name to be a link to GitHub. I updated the date to today and attributed it to @crspeller.